pixi.toml

```diff
@@ -0,0 +1,53 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: toml -*-
+
+#                             VVVVVV  minimum `pixi` version
+"$schema" = "https://pixi.sh/v0.36.0/schema/manifest/schema.json"
+
+[project]
+name = "napari-feedstock"
+version = "3.51.0"  # conda-smithy version used to generate this file
+description = "Pixi configuration for conda-forge/napari-feedstock"
+authors = ["@conda-forge/napari"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "win-64"]
```

Comment:
```suggestion
platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
```

?

Reply:
This file is auto-generated, we must not change it.

Reply:
Does that mean we need to change something else for arm64 packages?

Reply:
Nope. This is a helper file for local debugging (allows you to run `pixi run lint`) and things like that. The default platforms value for that field doesn't include `osx-arm64` in `conda-smithy`, that's all. We could fix it in `conda-smithy` if needed, but not here. Opened https://github.com/conda-forge/conda-smithy/issues/2357.